src/interfaces/artifact.ts

```diff
@@ -1,4 +1,6 @@
-import { AtrifactType } from '../enums/artfiact';
+import { Artifact3DType, ArtifactDEMType, ArtifactRasterType } from '@map-colonies/types';
+
+declare type AtrifactType = ArtifactDEMType | ArtifactRasterType | Artifact3DType;
 
 export interface Artifact {
```

Comment:
Shouldn't "Artifact" be a base interface for all domain "Artifacts"?

Reply:
no, 'Artifact' contains 'type' field, thats why the called  ArtifactType